radiation/python/radsw/cldprop_gt4py.py

```diff
@@ -757,6 +764,6 @@ def cldprop(
     valdict[var] = serializer.read(
         var, serializer.savepoint["swrad-cldprop-output-000000"]
     )
-    outdict_np[var] = outdict_gt4py[var].view(np.ndarray).squeeze()
+    outdict_np[var] = outdict_gt4py[var][:, :, 1:, ...].view(np.ndarray).squeeze()
```

Comment:
was this a buffering issue?

Reply:
Yes, I ended up making all fields size `nlp1` in the K dimensions for simplicity